src/core/evaluator.js

```diff
@@ -2421,7 +2426,19 @@ class PartialEvaluator {
         rotate += currentTransform[1] > 0 ? 90 : 270;
```

Comment:
I'm guessing this line might be connected to the test failures, since it no longer does the right thing with the new default value.